core/services/editor/EditorBlockAssetRegister.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+namespace EventEspresso\core\services\editor;
+
+use EventEspresso\core\domain\DomainInterface;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\services\assets\AssetRegisterInterface;
+use EventEspresso\core\services\assets\Registry;
+
+defined('EVENT_ESPRESSO_VERSION') || exit;
+
+
+
+/**
+ * Class EditorBlockAssetRegister
+ * Abstract parent for classes that handle asset registration for one or more editor blocks that share the same assets
+ *
+ * @package EventEspresso\core\services\editor
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+abstract class EditorBlockAssetRegister implements AssetRegisterInterface
```

Comment:
ya I initially didn't like the idea of **just** using "block" to reference these things because it just seems way too generic, but that said, we are using namespacing which pretty much negates that issue, and "block" is what WP uses, so ya... I'll change all usages of "EditorBlock" to just "Block"